homeassistant/components/islamic_prayer_times/sensor.py

```diff
@@ -50,9 +51,8 @@ def unique_id(self):
     @property
     def native_value(self):
         """Return the state of the sensor."""
-        return self.client.prayer_times_info.get(self.sensor_type).astimezone(
-            dt_util.UTC
-        )
+        timezone = datetime.now().tzinfo
+        return self.client.prayer_times_info.get(self.sensor_type).astimezone( timezone )
```

Comment:
This has no use, Home Assistant will convert it back to UTC again.

I'm not seeing the issues you are trying  to solve here?

Reply:
sorry, appears to have been a misconfig at my end. trying to replicate the issue to confirm